products/office.md

```diff
@@ -38,39 +52,60 @@ releases:
 
 -   releaseCycle: "2013"
     releaseLabel: "2013 SP1"
-    releaseDate: 2014-02-25
+    releaseDate: 2013-01-09
```

Comment:
Given support period overlap for 2013 and 2013 SP1 release, shouldn't we document those in separate entries ?

Reply:
Since both releases are EOL, it should be okay to collapse it into a single row.

Reply:
Thanks for the feedback. [MS](https://learn.microsoft.com/lifecycle/products/microsoft-office-2013) sees these service packs as separate cycles:
> Original Release -> Start: Jan 9, 2013,	End: Apr 14, 2015
> Service Pack -> Start: 1	Feb 25, 2014,	End: Apr 11, 2023

We should add them also as additional cycles as we have it like in Windows with 23H2, etc. Will do that and update this PR.

Reply:
> Since both releases are EOL, it should be okay to collapse it into a single row.

This may not be very practical for API users because of the lack of consistency. I think we should either document major versions, or minor versions, but not mix both.